my-app/src/components/employees/EmployeesItem.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+
+class EmployeesItem extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      item: props?.item,
+    };
+  }
+
+  render() {
+    const { item } = this.state;
+
+    return (
+      <li className="">
+        <h2 className="">{item?.username}</h2>
+        <p className="">{item?.email}</p>
+        <button className="">Delete</button>
+        <button className="">Edit</button>
```

Comment:
Why do you need buttons if they make no functionality?

Reply:
Max told us to add them, but we'll add functionality to them on later stages